JournalMonitor/JournalMonitor.cs

```diff
@@ -266,7 +264,7 @@ public static List<Event> ParseJournalEntry(string line, bool fromLogLoad = fals
                                         factions = getFactions(factionsVal);
                                     }
 
-                                    events.Add(new LocationEvent(timestamp, systemName, x, y, z, systemAddress, body, bodyType, docked, station, stationtype, marketId, allegiance, faction, economy, economy2, government, security, population, longitude, latitude, factions) { raw = line, fromLoad = fromLogLoad });
+                                    events.Add(new LocationEvent(timestamp, systemName, x, y, z, systemAddress, body, bodyType, docked, station, stationtype, marketId, allegiance, faction.name, faction.FactionState, economy, economy2, government, security, population, longitude, latitude, factions) { raw = line, fromLoad = fromLogLoad });
```

Comment:
Consider passing faction as an object? We're reconstituting the object at EDDI.cs line 926. We can simplify if we build the object here and simply pass the object.